src/main/java/org/entando/entando/aps/system/services/storage/LocalStorageManager.java

```diff
@@ -113,6 +113,18 @@ public void createDirectory(String subPath, boolean isProtectedResource) {
 		});
 	}
 
+	@Override
+	public boolean isDirectory(String subPath, boolean isProtectedResource) {
+		Boolean isDir = withValidResourcePath(subPath, isProtectedResource, (basePath, fullPath) -> {
```

Comment:
strange indentation

Reply:
I just double checked  of code indentation, and everything is properly indented